src/uwtools/tests/drivers/test_chgres_cube.py

```diff
@@ -27,10 +26,10 @@ def cycle():
 
 
 @fixture
-def config_file(tmp_path):
+def config(tmp_path):
```

Comment:
Here and below, since all drivers now accept a `dict` value for the constructor argument `config`, I removed the code that was YAML-dumping these `dict` objects to files, to eliminate some unnecessary file IO from the unit tests. Where possible, the driver test objects are instantiated with in-memory `dict` objects instead.

Reply:
Do we have a test that remains to ensure that the file option is tested?

Reply:
I just updated a test in `test_driver.py` to exercise this case. Good call, thanks.